Consul.AspNetCore.Test/Consul.AspNetCore.Test.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>netcoreapp3.1;net5.0;net6.0</TargetFrameworks>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```

Comment:
removes warnings about net5.0 being out of support

Reply:
ok, maybe we should remove these 2 unsupported targets at some point.